packages/react-scripts/package.json

```diff
@@ -53,40 +55,34 @@
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-testing-library": "^3.9.2",
-    "eslint-webpack-plugin": "^2.5.2",
-    "file-loader": "6.1.1",
-    "fs-extra": "^9.0.1",
-    "html-webpack-plugin": "4.5.0",
+    "eslint-webpack-plugin": "^2.5.4",
+    "file-loader": "^6.2.0",
```

Comment:
`file-loader` is no longer necessary in Webpack 5, you should instead use [Asset Modules](https://webpack.js.org/guides/asset-modules/)

Reply:
@dawnofmidnight I agree but did not find a way, resolving this comment in favor of the thread below related to this.